_data/jobs.yml

```diff
@@ -1,3 +1,8 @@
+- expires: 2022-10-31
+  location: Enthought, Austin, TX
+  name: Scientific Software Developer
+  posted: 2022-08-22
+  url: https://jobs.lever.co/enthought/b0130e7e-ca9a-4e89-9233-41b54efd292a?lever-origin=applied&lever-source%5B%5D=US-RSE
```

Comment:
Could just be https://jobs.lever.co/enthought/b0130e7e-ca9a-4e89-9233-41b54efd292a though I'm not sure it matters

Reply:
maybe better to keep the way it is, as they might use this to track our impact